src/status_im/ui/screens/privacy_and_security_settings/views.cljs

```diff
@@ -7,13 +7,20 @@
             [status-im.ui.components.react :as react]
             [status-im.multiaccounts.biometric.core :as biometric]
             [status-im.ui.components.topbar :as topbar]
-            [status-im.utils.platform :as platform])
+            [status-im.utils.platform :as platform]
+            [status-im.ui.screens.privacy-and-security-settings.events :as events])
   (:require-macros [status-im.utils.views :as views]))
 
+(defn separator []
```

Comment:
I think you could also use the component that you added into quo here?

Reply:
good idea :D